{"uuid":"1cb7c3f6-05e5-489c-bf97-0595c31adeea","historyId":"bf755c4b2a73e5b5ec64ee442a7abcff","testCaseId":"a46bf90c4d347c37965357d8dd1fd7b3","fullName":"src/test/resources/features/DeleteBook.feature:10","labels":[{"name":"tag","value":"DELETEBook"},{"name":"host","value":"laptop"},{"name":"thread","value":"2268@laptop.main(1)"},{"name":"feature","value":"Delete a book from the system"},{"name":"story","value":"User attempts to delete a non-existing book"},{"name":"suite","value":"Delete a book from the system"},{"name":"testClass","value":"User attempts to delete a non-existing book"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///E:/IT%20%20NOTES/IS3440%20-%20IT%20Quality%20Assurance/Group%2029/API_Testing/src/test/resources/features/DeleteBook.feature"},{"name":"package","value":"src.test.resources.features.DeleteBook_feature.Delete a book from the system"}],"links":[],"name":"User attempts to delete a non-existing book","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Unexpected status code! This is a BUG. expected [403] but found [404]","trace":"java.lang.AssertionError: Unexpected status code! This is a BUG. expected [403] but found [404]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\r\n\tat org.testng.Assert.assertEquals(Assert.java:132)\r\n\tat org.testng.Assert.assertEquals(Assert.java:1419)\r\n\tat org.testng.Assert.assertEquals(Assert.java:1383)\r\n\tat com.group_29.api_testing.stepDefinitions.deleteApiStepDefinitions.DeleteBooksTest.iShouldReceiveAResponseWithStatusCode(DeleteBooksTest.java:74)\r\n\tat ✽.I should receive a delete response with status code 403(file:///E:/IT%20%20NOTES/IS3440%20-%20IT%20Quality%20Assurance/Group%2029/API_Testing/src/test/resources/features/DeleteBook.feature:13)\r\n"},"stage":"finished","steps":[{"name":"Given I am a user in delete","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1735708015763,"stop":1735708015764},{"name":"When I send a DELETE request to \"/api/books/{id}\" with a non-existing book ID as a user","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1735708015764,"stop":1735708015930},{"name":"Then I should receive a delete response with status code 403","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Unexpected status code! This is a BUG. expected [403] but found [404]","trace":"java.lang.AssertionError: Unexpected status code! This is a BUG. expected [403] but found [404]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\r\n\tat org.testng.Assert.assertEquals(Assert.java:132)\r\n\tat org.testng.Assert.assertEquals(Assert.java:1419)\r\n\tat org.testng.Assert.assertEquals(Assert.java:1383)\r\n\tat com.group_29.api_testing.stepDefinitions.deleteApiStepDefinitions.DeleteBooksTest.iShouldReceiveAResponseWithStatusCode(DeleteBooksTest.java:74)\r\n\tat ✽.I should receive a delete response with status code 403(file:///E:/IT%20%20NOTES/IS3440%20-%20IT%20Quality%20Assurance/Group%2029/API_Testing/src/test/resources/features/DeleteBook.feature:13)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1735708015931,"stop":1735708015931}],"attachments":[],"parameters":[],"start":1735708015763,"stop":1735708015932}